yaml/service.go

```diff
@@ -118,11 +118,8 @@ func (s *ServiceSlice) Validate(pipeline []byte) error {
 				return err
 			}
 
-			invalid = fmt.Errorf(
-				"%w: %s",
-				invalid,
-				fmt.Sprintf("no name provided for service:\n%s\n ", string(source)),
-			)
+			// nolint:cSpell // ignore line length
+			invalid = fmt.Errorf("%w: %s", invalid, fmt.Sprintf("no name provided for service:\n%s\n ", string(source)))
```

Comment:
I think this is the wrong linter for the `nolint`

```suggestion
			// nolint:lll // ignore line length
			invalid = fmt.Errorf("%w: %s", invalid, fmt.Sprintf("no name provided for service:\n%s\n ", string(source)))
```

Reply:
See other comment